### DISCUSSION TITLE
come_korean

### DISCUSSION THREAD

**Discussion by EeeasyCode** [Schweinepriester/github-profile-achievements#271] [Q&A] [answered]
## YOLO -> 코드리뷰 없이 pr을 merge하면 된다

추가 사항이나 변경사항은 댓글로 남겨주세요 
저희 멋진 한국인이 되어봅시다!

**Comment by haideralicodes** [comment, marked as answer]
Pair Extraordinaire -> must have a coauthored commit present in the merged pull request.
Quickdraw -> Issue or pull request should close within 5 minutes.
Starstruck -> repo must have at least 16 stars.
The Galaxy Brain - > Discussion should also use the Q&A feature. At this time, it is possible if there are more than 2 accepted
Open at least 2 Pull Shark -> pull requests.
YOLO -> You can merge pull requests without code review

**Reply by EeeasyCode** [reply]
Thank you for your comments 👍👍

**Reply by haideralicodes** [reply]
Please can you mark this as answer

**Reply by EeeasyCode** [reply]
Sure !!
